### ISSUE TITLE
[spike] Technical discovery on CST Evidence headers

### ISSUE THREAD

**Issue by sortizsh** [department-of-veterans-affairs/va.gov-team#47246]
Pulled from [original ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/41818)

per comment, revisit the naming of file types in this tool and determine a solution

## Description

List of additional evidence should be unique or uniquely described.

Related: https://github.com/department-of-veterans-affairs/va.gov-team/issues/41447 (Under "Should" fix)

![List of additional evidence should be unique or uniquely described](https://user-images.githubusercontent.com/14154792/169905053-5d1b9fe2-3873-4463-a074-e3a01da0f4b3.png)

## Tasks

- [ ] Determine technical feasibility of using content team's recommendation of [File type, X of Y] as the header e.g. "Statement in Support of Claim for PTSD, 1 of 2"
- [ ] Determine technical feasibility of content team's recommendation of changing instances of the STR acronym in dropdowns and headers to fully say "Service treatment record"
- [ ] Revisit the naming of file types in this tool with the content team @katherine-fung and determine a solution
